ChangeLog

```diff
@@ -113,6 +113,7 @@ Changes
    * Only run AES-192 self-test if AES-192 is available. Fixes #963.
    * Tighten the RSA PKCS#1 v1.5 signature verification code and remove the
      undeclared dependency of the RSA module on the ASN.1 module.
+   * Removed support for the library reference configuration for picocoin.
```

Comment:
Use tense consistent with rest of the ChangeLog (which appears to be command tense)

```diff
-Removed
+Remove
```